### DISCUSSION TITLE
Resizable navigation pane for Default2022 Presentation Style

### DISCUSSION THREAD

**Discussion by ritchiecarroll** [EWSoftware/SHFB#1000] [Ideas]
First off, just playing around today with the new Default2022 style - it looks so great!!

In the older VS2013 presentation style, you could re-size the navigation pane - this was useful especially for verbose type names, like those with multiple generic types.

Wondering if this would be possible to add to the new theme...

Thanks,
Ritchie

**Comment by ritchiecarroll** [comment]
Created a pull request that does this for your review:
https://github.com/EWSoftware/SHFB/pull/1001

**Comment by EWSoftware** [comment]
Thanks. I merged the changes manually and made a few updates. I added a new ResizableTocColumn transformation argument that allows you to enable the resizable TOC column. I also moved the hard-coded styles to CSS classes.
